README.md

```diff
@@ -288,6 +288,9 @@ This repository provides two distinct ways for building and packaging tezos bina
 Read [the article](./docs/baking.md) to find out an easy way to set up
 baking instance on Ubuntu using packages provided by our launchpad PPA.
 
+You can also use our interactive CLI utility, `baking_wizard`, that comes
```

Comment:
I was going to comment this same thing.

One more addition is that since we'll have to make this update later it would be nice for this PR not to close the issue.